ui/.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-cd ./ui && npx lint-staged
```

Comment:
How should we handle this file? @g1eny0ung

Reply:
Don‘t delete this file, this is the hook used to format the TS/JS code.

Reply:
Sorry. It's a mistake 😢